medvis/dicom/readme.md

```diff
@@ -1 +1,20 @@
-Adds data reading support for the DICOM image/volume file format (.dcm file ending). Uses grassroots dicom library for loading.
+# Inviwo DICOM module
+
+Adds data reading support for the DICOM image/volume file format (.dcm file ending). Uses the Grassroots DICOM library [GDCM](https://github.com/malaterre/GDCM) for reading DICOM files and directories.
+
+When using vcpkg, set the CMake option `IVW_USE_EXTERNAL_GDCM` to false and add `dicom` to the `VCPKG_MANIFEST_FEATURES` as in the following CMake preset:
+
+```json
```

Comment:
CMakePresets can be included
https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#id5
So, this could be moved to a CMakePresets file in this directory and then we can change the instructions to include the file and inherit dicom 
```suggestion

```json
“include”: [${sourceParentDir}/modules/medvis/dicom/CMakePresets.json]
```

Reply:
that will not work well here since the flags will not compose so you have to add to them your self.